llvm/test/CodeGen/AIE/aie2/schedule/interblock/liveness.mir

```diff
@@ -0,0 +1,229 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+
+# RUN: llc --mtriple=aie2 --run-pass=postmisched --issue-limit=1 -debug-only=machine-scheduler %s -o - 2>%t.log
+# RUN: cat %t.log | FileCheck %s --check-prefix=CHECK-LIVENESS
+# REQUIRES: asserts
+
+# These tests verify liveouts of every MachineBasicBlock in MachineFunction and 
+# liveins of every Region in MachineBasicBlock.
```

Comment:
Could you add tests with function calls? Those are important to test for two reasons:

- They spilt a BB in multiple sched regions
- They come with a regmask operand like `csr_aie2` which clobbers most registers, including reserved ones